docs/settings.md

```diff
@@ -0,0 +1,78 @@
+## Pageserver
+
+### pg_listen_addr
```

Comment:
That is `listen-pg` and the next one is `listen-http`, according to the code: 
https://github.com/zenithdb/zenith/blob/431d32756bf1bf2fa1be4aaf29e323de91fd5780/pageserver/src/bin/pageserver.rs#L285

I would adjust the readme to reflect the code, but lmk if the reverse operation is safe to do and desired instead.

Reply:
Ah yeah, the option names in the config file, on command line, and in the PageServerConf don't match up exactly. It would probably make most sense to use the names as they are in the config file. That would be "listen_pg_addr" and "listen_http_addr".

This will get better with https://github.com/zenithdb/zenith/pull/871.